server/src/bots/telegram/services/registry/getParameterAfterCommandFromMessage.test.ts

```diff
@@ -0,0 +1,90 @@
+import * as getInfoMessage from '../../../../utils/getErrorMessages';
+import { getParameterAfterCommandFromMessage } from './getParameterAfterCommandFromMessage';
+import { logger } from '../../../../utils/logger';
+
+const singleParameterAfterCommandMock = [
```

Comment:
Could use less of co words here, same as country flags.
I'ma asking cause from my point of view Unit test's should not depend on concrete constants used in the system, but rather should verify concrete concepts working.
So I'd actually decrease amount of words. here., and leave only those which are different by concept.